x-pack/legacy/plugins/infra/server/lib/snapshot/snapshot.ts

```diff
@@ -63,6 +69,24 @@ export class InfraSnapshot {
   }
 }
 
+// This is used by the getAllCompositeData to select the bucket to return.
```

Comment:
I love comments, but I'm not sure we need these given the parameters for `getAllCompositeData()` are fully typed and explain themselves?